haystack/pipelines/config.py

```diff
@@ -87,7 +87,7 @@ def get_component_definitions(
                         param_name,
                         name,
                         key,
-                        value,
+                        f"{value[:5]}...",
```

Comment:
```suggestion
                        "***",
```

I would completely hide the value to be fair, better safe than sorry.

Reply:
I think you must update the relative test too, or it's going to fail. 🤔

Reply:
Ah shoot, yes, will do.

Reply:
Updated.